qa/zcash/check-security-hardening.sh

```diff
@@ -16,8 +16,8 @@ function test_basic_hardening {
 }
 
 function test_fortify_source {
-    if "${REPOROOT}/qa/zcash/checksec.sh" --fortify-file "$1" | grep -q "FORTIFY_SOURCE support available.*Yes" &&
-       "${REPOROOT}/qa/zcash/checksec.sh" --fortify-file "$1" | grep -q "Binary compiled with FORTIFY_SOURCE support.*Yes"; then
+    if { "${REPOROOT}/qa/zcash/checksec.sh" --fortify-file "$1" | grep -q "FORTIFY_SOURCE support available.*Yes"; } &&
+       { "${REPOROOT}/qa/zcash/checksec.sh" --fortify-file "$1" | grep -q "Binary compiled with FORTIFY_SOURCE support.*Yes"; } then
```

Comment:
I think this line should strictly speaking end in ``}; then``. ``} then`` is accepted and apparently works, but I can find no documentation guaranteeing that it will work.